src/main/java/com/danielpsf/labs/field/statistic/metric/domain/MetricRequest.java

```diff
@@ -1,14 +1,19 @@
 package com.danielpsf.labs.field.statistic.metric.domain;
 
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Getter;
 
 import java.util.Date;
 
 @Getter
 @Builder
+@AllArgsConstructor
 public class MetricRequest {
 
     private Double vegetation;
     private Date occurrenceAt;
+
+    public MetricRequest() {
```

Comment:
This is necessary to avoid `jackson` to break due to no default constructor found while executing the post from the integration test.

No breaking changes were inserted on current functionality, or on the unit tests.

**Reference**:
- https://www.baeldung.com/jackson-exception (item 3)